### PR TITLE
Make namespaces aliases possible.

### LINKED ISSUE #113: Symfony 4 and wrong path to Entities
Symfony 4 updates file structure and removes AppBundle folder. 

The easiest solution would be to remove AppBundle from path and change default namespace from AppBundle to App.

### PR DESCRIPTION
**Pull request by jlagneau**
Add the possibility to alias namespaces to a different path (pretty much like composer do to resolve namespace for autoloading).

Allow the possibility to fix #113 by creating a simple alias.

Eg:
```yml
namespaces:
    alias:
        - { namespace: App\, resolvePath: src/ }
    entity: App\Entity

# etc...
```
Running the command `php vendor/bin/schema generate-types . config/schema.yaml` would correctly create entities, enum and interfaces in the correct path and keep their correct namespace.

Let me know if there's a more elegant way to do this.

### CODE REVIEW

**Comment by jlagneau** [comment]
I've added the possibility to resolve empty namespaces for retro compatibility (and to reflect how composer handle psr-4 autoloading resolution):

```yml
namespaces:
    alias:
        - { namespace: ~, resolvePath: src/ }
    entity: AppBundle\Entity
# etc...
```
With the command `php vendor/bin/schema generate-types . app/config/schema.yaml`, the `AppBundle` namespace would now be resolved in `src/AppBundle` for the files generation.

**Comment by Saphyel** [comment]
Looks good to me! 

I'm think to use your branch as stable version until this is merge.
Thanks for your time! ^^

File: src/TypesGeneratorConfiguration.php [hunk 1]
```diff
@@ -92,6 +92,23 @@ function ($rdfa) {
                     ->addDefaultsIfNotSet()
                     ->info('PHP namespaces')
                     ->children()
+                        ->arrayNode('alias')
```
Comment (dunglas) [inline src/TypesGeneratorConfiguration.php]: Can't we just add a `prefix` key for each namespace?

Comment (jlagneau) [reply src/TypesGeneratorConfiguration.php]: A prefix would work too I guess, I just tried to mimic the way composer.json configuration handled that. Should we have multiple prefixes or just one would do?

**Comment by dunglas** [comment]
Thank you very much for handling this @jlagneau. I included your commits in #121.

**Comment by giorgioma** [comment]
I am getting an "Unrecognized option alias" error.
Were these change overwritten by mistake in [this commit](https://github.com/api-platform/schema-generator/commit/04e337f6fefd8608494b21e3afd0a3b781332d94) and [this PR](https://github.com/api-platform/schema-generator/pull/121)?

**Pull request closed without merging**